lib/fxpay/init_module.js

```diff
@@ -0,0 +1,38 @@
+// This is a minimal bootstrapping script that defines the fxpay
+// namespace and allows all other modules to require fxpay attributes.
+(function() {
+  'use strict';
+
+  if (typeof window.fxpay !== 'undefined') {
+    throw new Error('fxpay/init_module.js must be loaded before other scripts');
+  }
+
+  var exports = window.fxpay = {};
+
+  exports.getattr = function getattr(attr) {
```

Comment:
As I mentioned on the call would be nice to maybe make use of `obj.hasOwnProperty(prop)` here so we're not looking up things down the prototype chain.